### ISSUE TITLE
 Google Dataset Search

### ISSUE THREAD

**Issue by cgars** [G-Node/gogs#19]
fwd: 

> https://toolbox.google.com/datasetsearch
> 
> Is it possible to prepare the GIN repo pages or DOI landing pages to be included?
> 
> https://developers.google.com/search/docs/data-types/dataset
> 
> Best,
>   Thomas

**Comment by cgars** [comment]
Just as a side note. GIN is in google dataset search already (via datacite). However, you must search for [g-node](https://toolbox.google.com/datasetsearch/search?query=g-node&docid=VDWOP7wBzYphrYycAAAAAA%3D%3D) because that's what we register everything under (also true for PLOS)... Arguably the bigger step for visibility under the GIN name is is actually using that term (or rebranding GIN as gnode)

**Comment by achilleas-k** [comment]
I did a few searches for specific titles that appear on our DOI listing. It looks like the most recent ones return no results, but anything from last year turns up fine. I wonder why that might be (this is just an aside and personal curiosity).

Regardless, as you said it would be much better for visibility if the datasets linked directly to the GIN landing page instead of datacite.

**Comment by cgars** [comment]
> Regardless, as you said it would be much better for visibility if the datasets linked directly to the GIN landing page in addition to datacite.

I think it would be more important that they are registered under GIN as the data provider and not g-node... . I am not sure whether redundant registering is the good way to go here. How often should the same data be registered? and at how many locations? Also at Bing data? Baidu? Yandex? . Even if they remove redundancy at Google you still have to provide the data like n times?

**Comment by achilleas-k** [comment]
Well I would prefer they linked to GIN only and not datacite, but I figured removing datacite results wouldn't possible. I just meant to say that the goal should be that if a search turns up a dataset registered at the G-Node, the primary link should be to doid.gin.g-node.org.

**Comment by achilleas-k** [comment]
> After all the data is registered with them!
Right, OK, fair point.

**Comment by twachtler** [comment]
I agree that it would be desirable to get a link to the repo rather than to a datacite page where you have to click again to get to the dataset landing page.  The issue is how Google displays the results - shouldn't it be easy for them to include the doi, which would always be the most reliable way to get to the data?

But anyway if it takes a year before the datacite information becomes available to Google, then it would be worthwhile enhancing the GIN pages so that Google can grab the information directly.

The datacenter registered with DataCite is G-Node not GIN and that's ok, we want to issue DOIs for abstracts on GCA, too. The point is not to search for all data on GIN but to have datasets on GIN turn up if they match the search.
